internal/cmd/export.go

```diff
@@ -231,7 +232,8 @@ kubectl pgo support export daisy --monitoring-namespace another-namespace --outp
 | PGO CLI Support Export Tool
 | The support export tool will collect information that is
 | commonly necessary for troubleshooting a PostgresCluster.
-| Note: No data or k8s secrets are collected.
+| Note: No data or k8s secrets are collected, however,
+| kubectl is used to list plugins on the user's machine.
```

Comment:
```suggestion
| Note: No data or k8s secrets are collected.
| However, kubectl is used to list kubectl plugins on the user's machine.
```
?
I just like the period after `No data or k8s secrets are collected.` It's a definitive statement.
Kubeectl to list kubectl plugins is redundant, but I want to be redundant here.

Reply:
I think you're right. I was internally debating it.